path_data_test.go

```diff
@@ -182,9 +182,9 @@ func TestVersionedKV_Data_Put(t *testing.T) {
 	}
 
 	events.expectEvents(t, []expectedEvent{
-		{"kv-v2/metadata-write", "metadata/foo"},
-		{"kv-v2/data-write", "data/foo"},
-		{"kv-v2/data-write", "data/foo"},
+		{"kv-v2/metadata-write", "foo"},
+		{"kv-v2/data-write", "foo"},
+		{"kv-v2/data-write", "foo"},
```

Comment:
This diff doesn't look right to me - any downstream consumers who want to re-fetch the secret will need `data/` in the path. How come it's been deleted?

Reply:
Updated based on our discussions.